src/routes/v2/authenticated/sites.ts

```diff
@@ -128,7 +128,6 @@ export class SitesRouter {
 
     router.get(
       "/",
-      this.statsMiddleware.countGithubSites,
```

Comment:
Removing the countGithubSites middleware to prevent excessive calls to github

cc @isomerpages/iso-engineers